### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [yxzlwz/blog#2] [Announcements]
# about/

你好，我是异想之旅
 热爱骑行和听歌的高中生，全栈开发&信竞选手，偶尔客串情感大师。 异想之旅最早是2018年4月1日我随手注册的公众号 名字的由来记不得了，好像最开始想叫奇幻之路，只是由于重名，不断更改的过程中碰上了这个“无人认领”的名字 那时这甚至还是一个声音频道，也不知什么时候成为了我的技术分享？ 1685705610785 2020年网课期间，...

http://blog.yixiangzhilv.com/about/

**Comment by ghost** [comment]
好厉害！加油加油！

**Comment by soyisou** [comment]
强
